docs/quickstart.rst

```diff
@@ -0,0 +1,601 @@
+.. eql:section-intro-page:: rocket
+
+==========
+Quickstart
+==========
+
+Welcome to EdgeDB!
+
+This quickstart will walk you through the entire process of creating a simple EdgeDB-powered application: installation, defining your schema, adding some data, and writing your first query. Let's jump in!
+
+
+..    :ref:`Installation <ref_quickstart_install>`
+..    :ref:`Initialize a project <ref_quickstart_createdb>`
+..    :ref:`Set up your schema <ref_quickstart_createdb_sdl>`
+..    :ref:`Insert data <ref_quickstart_insert_data>`
+..    :ref:`Run some queries <ref_quickstart_queries>`
+..    :ref:`Migrate your schema <ref_quickstart_migrations>`
+..    :ref:`Computables <ref_quickstart_computables>`
+..    :ref:`Onwards and upwards <ref_quickstart_onwards>`
+
+.. _ref_quickstart_install:
+
+1. Installation
+===============
+
+First let's install the EdgeDB CLI. Open a terminal and run the appropriate command below.
+
+**macOS/Linux**
+
+.. code-block:: bash
+
+    $ curl https://sh.edgedb.com --proto '=https' -sSf1 | sh
+
+**Windows**
+
+.. code-block::
+
+    # in Powershell
+    PS> iwr https://ps1.edgedb.com -useb | iex
+
+This command downloads and executes a bash script that installs the ``edgedb`` CLI on your machine. You may be asked for your password. Once the installation completes, restart your terminal so the ``edgedb`` command becomes available.
+
+Now let's use the CLI to install the latest version of the EdgeDB database. (The CLI is separate from the database itself.)
+
+.. code-block:: bash
+
+    $ edgedb server install
+
+
+> Depending on your OS, the native server packages might not yet be available, and you might need to install and run Docker to complete the EdgeDB server installation.
```

Comment:
What's `>` is used for?

Reply:
I was using it to indicate that the paragraph should be an inline block. The docs renderer currently doesn't implement inline blocks so indenting normally was throwing an error. Perhaps @jaclarke could try to fix this?

Reply:
```
[docs (dev)] Error: Command failed: ./.venv/bin/sphinx-build -b xml -q -D master_doc=index -W -n /var/folders/3g/lrw7ksg134v9vgn45llklcb40000gn/T/tmp-68251-S6BZhsXnkTKh .build-cache/_xml/docs

Exception occurred:
  File "/Users/colinmcd94/Documents/edgedb/ui/apps/edgedb.com/.repos/edgedb/edb/tools/docs/__init__.py", line 40, in apply
    raise shared.EdgeSphinxExtensionError(
edb.tools.docs.shared.EdgeSphinxExtensionError: blockquote found: '<block_quote><paragraph>Quick note! You can have several &quot;instances&quot; of EdgeDB running on your\ncomputer simultaneously. Each instance contains several &quot;databases&quot;. Each\ndatabase may contain several &quot;schema modules&quot; (though commonly your schema\nwill be entirely defined inside the <literal>default</literal> module).</paragraph></block_quote>'
The full traceback has been saved in /var/folders/3g/lrw7ksg134v9vgn45llklcb40000gn/T/sphinx-err-j_5pew1i.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!

    at ChildProcess.exithandler (child_process.js:308:12)
    at ChildProcess.emit (events.js:315:20)
    at ChildProcess.EventEmitter.emit (domain.js:486:12)
    at maybeClose (internal/child_process.js:1048:16)
    at Socket.<anonymous> (internal/child_process.js:439:11)
    at Socket.emit (events.js:315:20)
    at Socket.EventEmitter.emit (domain.js:486:12)
    at Pipe.<anonymous> (net.js:673:12) {
  killed: false,
  code: 2,
  signal: null,
  cmd: './.venv/bin/sphinx-build -b xml -q -D master_doc=index -W -n /var/folders/3g/lrw7ksg134v9vgn45llklcb40000gn/T/tmp-68251-S6BZhsXnkTKh .build-cache/_xml/docs'
}
q^C%                                      
```

Reply:
Done